providers/airbyte/provider.yaml

```diff
@@ -25,6 +25,7 @@ state: ready
 source-date-epoch: 1744788595
 # note that those versions are maintained by release manager - do not update them manually
 versions:
+  - 5.0.3
```

Comment:
please revert this change.
changes to versions is done only by a request from a release manager

Reply:
@eladkal should I revert the all places, correct?